tests/Doctrine/Deprecations/DeprecationTest.php

```diff
@@ -202,6 +202,23 @@ public function testDeprecationWithIgnoredPackage(): void
         $this->assertEquals(['https://github.com/doctrine/orm/issue/1234' => 1], Deprecation::getTriggeredDeprecations());
     }
 
+    public function testDeprecationWithIgnoredLink(): void
+    {
+        Deprecation::enableWithTriggerError();
+        Deprecation::ignoreDeprecations('https://github.com/doctrine/orm/issue/1234');
+
+        Deprecation::trigger(
+            'doctrine/orm',
+            'https://github.com/doctrine/orm/issue/1234',
+            'this is deprecated %s %d',
+            'foo',
+            1234
+        );
+
+        $this->assertEquals(1, Deprecation::getUniqueTriggeredDeprecationsCount());
```

Comment:
:thinking: if the deprecation is ignored, why is it taken into account in the count?

Reply:
It's the same test as the ane above for ignore by package. I don't know the reason behind this.

Reply:
This test does not make sense to me, and in fact, the existing code makes no sense to me either.

https://github.com/doctrine/deprecations/blob/ebe09b4ef27da986f0f19b88715d100dbaf86ea3/lib/Doctrine/Deprecations/Deprecation.php#L55-L59

Why is `ignoredLinks` a hash of counters instead of a hash of booleans?